src/themes/dashboard/SampleReactTable.js

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo } from 'react';
+import { Heading, Text, Table, Thead, Tbody, Tr, Th, Td, chakra } from '@chakra-ui/react';
+import { TriangleDownIcon, TriangleUpIcon } from '@chakra-ui/icons';
+import { useTable, useSortBy } from 'react-table';
+
+const SampleReactTable = () => {
+  const data = React.useMemo(
```

Comment:
this doesn't really matter but you did `React.useMemo` here and `useMemo` below, best to pick one and stick to it (I'm indifferent)

Reply:
Whoops, yeah this was definitely by accident 😅